### DISCUSSION TITLE
[BE] 패키지 구조

### DISCUSSION THREAD

**Discussion by sangwonsheep** [Kernel360/F2-BAGUNI#80] [메모장]
- dto의 경우 model 하위에 위치하는 것이 아닌 service 하위에 위치해야 합니다.
  - dto는 repository에 연관되는 것이 아닌, service에 연관되어 사용되기 때문입니다.

link의 경우 model/provider 디렉토리 안에 Provider 클래스가 위치하고 있습니다.
tag의 경우 model 디렉토리 안에 Provider 클래스가 위치하고 있습니다.
둘 중 한 가지 방향으로 잡고 가야 할 것 같습니다.

**Comment by kimminkyeu** [comment]
해결 완료 -> #82 PR 에서 구조 변경했습니다.
